web/impact/impact/settings.py

```diff
@@ -199,6 +200,18 @@ class Base(Configuration):
         'simpleuser.email_model_backend.EmailModelBackend',
         'django.contrib.auth.backends.ModelBackend',
     )
+    JWT_AUTH = {
+        'JWT_ALLOW_REFRESH': True,
+        'JWT_EXPIRATION_DELTA': datetime.timedelta(days=7),
```

Comment:
wondering if this window is too long as a default. I was thinking a few hours seems more reasonable.

Reply:
The reason we settled on this value is that I believe it will effect how long users remain logged into accelerate.  Currently I don't believe there is a time limit on that so we decided we wanted to make it as long as is reasonable.  A week between logins seems reasonable to me given what we have don't historically.  A few hours seems too short to me.